src/categories/controller/categories.controller.ts

```diff
@@ -25,16 +29,33 @@ import { Role } from 'src/auth/roles.enum';
 import { CategoryDto } from '../dtos/response/category.dto';
 import { CategoriesService } from '../service/categories.service';
 import { CreateCategoryDto } from '../dtos/request/create-category.dto';
+import { PaginationCategoryDto } from '../dtos/response/pagination-categories.dto';
 
 @ApiTags('Categories')
 @Controller('categories')
 export class CategoriesController {
   constructor(private readonly categoriesService: CategoriesService) {}
 
   @Get('all')
```

Comment:
Why did you use the `/categories/all` instead of simple `/categories`?

Reply:
Because I didn't update  a prefix in the endpoint  after adding the categories pagination and now I already removed path prefix  and used a simple `/categories`